module/localisation/FieldLocalisationNLopt/data/config/FieldLocalisationNLopt.yaml

```diff
@@ -0,0 +1,74 @@
+# Controls the minimum log level that NUClear log will display
+log_level: DEBUG
+
+# Size of each cell in the occupancy map, each cell has an area of grid_size x grid_size [metres]
+grid_size: 1e-2
+
+# Starting side of the field looking towards own goal from centre of field (LEFT, RIGHT, EITHER or CUSTOM)
+starting_side: LEFT
```

Comment:
Can we have this as `EITHER`?

Reply:
Should work on both, I have a check on startup for which side results in best fit